admin/config/locales/en.yml

```diff
@@ -1718,6 +1718,8 @@ en:
         last_name: Last Name
         last_order: Last Order
         last_order_at: Last Order At
+        last_logged_in_at: Last Logged In At
```

Comment:
dunno if this should really have the "At" on the end, might read a little weird..

Reply:
I'd say drop the `at`